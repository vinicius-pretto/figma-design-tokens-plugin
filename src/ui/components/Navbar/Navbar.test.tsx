import * as React from "react";
import { render } from "@testing-library/react";
import NavBar from "./Navbar";

describe("Navbar", () => {
  it("snapshot", () => {
    const { asFragment } = render(<NavBar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
