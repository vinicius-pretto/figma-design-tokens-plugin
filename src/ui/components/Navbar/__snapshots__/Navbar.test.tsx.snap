// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
  >
    <ul>
      <li>
        <a
          class="section-title"
          href="#"
        >
          Tokens
        </a>
      </li>
      <li>
        <a
          class="label"
          href="#"
        >
          CSS
        </a>
      </li>
      <li>
        <a
          class="label"
          href="#"
        >
          SCSS
        </a>
      </li>
      <li>
        <a
          class="label"
          href="#"
        >
          JSON
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
